### PR TITLE
SONAR-7103 new_debt_ratio not based on new_technical_debt on non-file

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/computation/sqale/SqaleNewMeasuresVisitor.java [hunk 1]
```diff
@@ -203,18 +214,28 @@ private static boolean isLineInPeriod(long lineDate, Period period) {
   }
 
   public static final class NewDevelopmentCostCounter {
```
Comment (julienlancelot) [inline server/sonar-server/src/main/java/org/sonar/server/computation/sqale/SqaleNewMeasuresVisitor.java]: Name of the counter should be renamed

Comment (sns-seb) [inline server/sonar-server/src/main/java/org/sonar/server/computation/sqale/SqaleNewMeasuresVisitor.java]: true

**Comment by julienlancelot** [comment]
LGTM !

**Comment by dbolkensteyn** [comment]
woohoo, merge & deploy? :)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'protobuf' package (org.sonarsource.sonarqube:sonar-db:target/generated-sources/protobuf/org/sonar/db/protobuf) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Pull request merged**